components/auth-proxy-service/src/main/java/com/wso2telco/proxy/attributeshare/ProvisionScope.java

```diff
@@ -0,0 +1,34 @@
+package com.wso2telco.proxy.attributeshare;
+
+import org.wso2.carbon.identity.application.authentication.framework.exception.AuthenticationFailedException;
+
+
+/**
+ * Created by aushani on 8/31/17.
+ */
+public class ProvisionScope extends  AbstractAttributeShare{
+    
+
+    @Override
+    public void mandatoryFeildValidation() {
+      /*mandatory feild validation in next phase*/
+    }
+
+    @Override
+    public void scopeNClaimMatching() {
+     /*
+     * this should be implemented in the next phase*/
+    }
+
+    @Override
+    public void shaAlgortithemValidation() {
+        /*
+     * this should be implemented in the next phase*/
+    }
+
+    @Override
+    public String attShareDetails(String operatorName, String clientId,String loginhintMsisdn,String msisdn) throws AuthenticationFailedException {
+
+        return getTrsutedStatus(operatorName, clientId,loginhintMsisdn,msisdn);
```

Comment:
If no specific implementation other than the super method no need to override since super method is public

Reply:
Fixed